### PR TITLE
add test to script

### PR DESCRIPTION
**Pull request by penglerts**

### CODE REVIEW

File: cli/setup_wif.py [hunk 1]
```diff
@@ -205,14 +221,15 @@ def main():
     parser.add_argument("--pool_name", help="Custom workload identity pool name", default="splunk-identity-pool")
     parser.add_argument("--provider_name", help="Custom workload identity provider name", default="splunk-provider")
     parser.add_argument("--dry_run", help="Just print what commands will be executed", action="store_true")
+    parser.add_argument("--no_interactive", action="store_true")
 
     project_ids = []
-    args = parser.parse_args()
+    args = parser.parse_args(args)
```
Comment (bgola-signalfx) [inline cli/setup_wif.py:227]: nit: do we really need to pass `args` to the `parse_args` here?
```suggestion
    args = parser.parse_args()
```

File: cli/test/test_setup_wif.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
+import sys
+import requests
+import os
+import string
+import random
+import unittest
+from dotenv import load_dotenv
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```
Comment (bgola-signalfx) [inline cli/test/test_setup_wif.py:9]: Why do we need this? Can you add a comment here please?

**Review by bgola-signalfx — APPROVED** [review]
Great work!

File: cli/setup_wif.py (hunk 1 above)
Comment (penglerts) [reply cli/setup_wif.py:227]: yes, by default it take it from systeam args, and it is null, but from unit test we send it like that

File: cli/test/test_setup_wif.py (hunk 2 above)
Comment (penglerts) [reply cli/test/test_setup_wif.py:9]: moved test to same dir as a script and now it is not necessary

**Review by bgola-signalfx — APPROVED** [review]

**Pull request merged**